tests/calculations/test_autoinvalidate_cache.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""Test the automatic 'invalidates_cache' attribute for exit codes."""
+
+from __future__ import absolute_import
+import pytest
+
+import aiida
+from aiida.plugins import CalculationFactory
+from aiida.plugins.entry_point import get_entry_point_names
+
+if aiida.__version__ < '1.1':
```

Comment:
Is this string comparison always going to work?

Reply:
That works as long as AiiDA doesn't put stuff at the beginning of its `__version__` it will work - because `.` comes before the numbers.

Anyway, should I wrap this in a `distutils.version.LooseVersion` or `StrictVersion`, to make it safe?
The difference between the two here would be that ``LooseVersion('1.1.0b1') < LooseVersion('1.1')`` is ``False``, while ``StrictVersion('1.1.0b1') < StrictVersion('1.1')`` is ``True``.

Reply:
Since the feature was added in `1.1.0` I would do `StrictVersion('1.1.0')`? Since we won't be releasing any `1.1.0` alphas or betas this should work

Reply:
Done! Do you happen to know why `pylint` thinks there's no such thing as `distutils.version.StrictVersion`?

Reply:
Remember having encountered this problem before, but don't remember or at least don't know the fix anymore. Probably just an exclude statement